test/NuGet.Core.FuncTests/Dotnet.Integration.Test/ProjectFileBuilder.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+using Dotnet.Integration.Test;
+
+namespace NuGet.Test.Utility
+{
+    /// <summary>
+    /// Represents a builder for dotnet core projects for integration tests.
+    /// Relies on <c>MsbuildIntegrationTestFixture</c> to programmatically create the project
+    /// </summary>
+    /// <seealso cref="MsbuildIntegrationTestFixture"/>
+    public class ProjectFileBuilder
```

Comment:
this is only a test class, but still I consider it good practise to not make types public unless they need to be. This is critical in our projects that get packed into packages, as customers might take a dependency and therefore it limit our ability to refactor.

Reply:
My intention with this builder is to have some help when writing tests. So, I want this class to be public. The only reason I have this Builder here is because I don't want to add a dependency between Test.Utility and `Dotnet.Integration.Tests`.

The only place i need the `MsbuildIntegrationTestFixture` is when the project is created. I'm thinking of creating static project files to get rid of `MsbuildIntegrationTestFixture`

Reply:
> I'm thinking of creating static project files to get rid of MsbuildIntegrationTestFixture

I would discourage that. 

Static projects means, eventually you get to a point where the SDK being tested is not shipping with the targeting pack for that eventually old framework. 

That means one of following will happen: 
1. Test fails. 
1. Hit the network and try to download the targeting. (note these are 30-40mbs, and will be download for each test).

Reply:
Ok. I will make this class internal and leave it there.